src/main/java/cloud/fogbow/common/models/SystemUser.java

```diff
@@ -22,6 +24,14 @@ public String getIdentityProviderId() {
         return identityProviderId;
     }
 
+    public Set<String> getUserRoles() {
+    	return roles;
```

Comment:
The spacing seems to be different from the standard in other places of the code.

Reply:
Fixed.